core/src/script/CGXP/tools/autoProject.js

```diff
@@ -0,0 +1,91 @@
+/**
```

Comment:
This the class is named ``OpenLayers.AutoProjection`` you should consider renaming the file to ``AutoProjection.js`` and move it to ``openlayers.addins/``, don't you think?